SampleGallery/Samples/SDK 16299/LinearGradients/LinearGradients.xaml.cs

```diff
@@ -0,0 +1,263 @@
+﻿//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, 
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, 
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. 
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, 
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, 
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH 
+// THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+//*********************************************************
+
+using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Composition;
+using Windows.UI.Xaml.Hosting;
+using Windows.UI;
+using System.Numerics;
+
+namespace CompositionSampleGallery
+{
+    public sealed partial class LinearGradients : SamplePage
+    {
+        public LinearGradients()
+        {
+            this.InitializeComponent();
+        }
+
+        public static string StaticSampleName => "Linear Gradients";
```

Comment:
Fields should go at the start or end of the class, not in the middle. Currently they're between the constructor and the first private method.

Reply:
this is how the SampleTemplate is set up for all samples in the sample gallery. all the other samples follow this format and I did not modify this section aside from the names.